src/pages/patterns/read-only-states-pattern/index.mdx

```diff
@@ -0,0 +1,360 @@
+
+![Enabled state for component controls.](images/read-only-component-controls-enabled.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+#### Component icon signifiers
+
+Signifier icons included in a component’s enabled state, like chevron arrow
+icons, close icons, and calendar icons, should use the $icon-disabled color
+token.
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="Read-only">
+
+![Read-only state for icon signifiers.](images/read-only-component-icon-signifiers.png)
+
+</Tab>
+
+<Tab label="Enabled">
+
+![Enabled state for icon signifiers.](images/read-only-component-icon-signifiers-enabled.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+### Best practices
+
+#### State readability
+
+Don’t use a disabled state for components if they need to be read by the user.
+Unlike read-only states, disabled states are not read by screen readers and do
+not pass visual contrast, making them inaccessible if they need to be
+interpreted.
+
+#### Read-only viewports
+
+Do maintain a component’s disabled state when in a read-only view. Don’t change
+a component’s state from disabled to read-only because of an active read-only
+view; some states should remain disabled depending on the use case.
+
+#### Disabled versus read-only states
+
+Do use a disabled state when a component is temporarily unavailable pending user
+action. Don’t change a component’s state from disabled to a read-only state.
+Alternatively, don’t use a read-only state to replace a disabled state.
+
+<DoDontRow>
+<DoDont
+type="do"
+colMd={6} colLg={6}
+caption="Do use if the component is not interactable but should be read by the user.">
+
+![Do use if the component is not interactable but should be read by the user.](./images/read-only-do-1.png)
+
+</DoDont>
+<DoDont
+colMd={6} colLg={6}
+type="dont"
+caption="Don’t use disabled states in place of read-only states.">
+
+![Don’t use disabled states in place of read-only states.](./images/read-only-dont-1.png)
+
+</DoDont>
+</DoDontRow>
+
+### Interactions
+
+#### Mouse
+
+The arrow cursor reinforces that a component is in a read-only state and is not
+interactive. The cursor's state reflects a component's shift in focus from
+interactive to informative.
+
+<Row>
+<Column colLg={8}>
+
+![Read-only state arrow cursor.](/images/read-only-cursor.png)
+
+<Caption>Read-only state arrow cursor.</Caption>
+
+</Column>
+</Row>
+
+#### Keyboard
+
+Interactive operations included in a component’s enabled state should be removed
+or modified for its read-only state. The component should remain navigable with
+a keyboard.
+
+## Content
+
+Components should include the same content used in the component’s enabled
+state. However, when the content in the enabled state is instructive, like a
+dropdown with no current selection, the content may need to change to be
+informative.
+
+<Row>
+<Column colLg={8}>
+
+<Tabs>
+
+<Tab label="Read-only">
+
+![Read-only content for a dropdown with no selection.](images/read-only-content.png)
```

Comment:
In this image, the field text color should still pass 4.5:1 contrast, change to text-secondary?
<img width="878" alt="image" src="https://user-images.githubusercontent.com/11670886/206320734-527c0f2e-cdf8-4581-bbe1-8edbab84dcbd.png">

Reply:
Ok yes, I thought something was off here. I will try out `$text-secondary`.